resources/views/play/show.blade.php

```diff
@@ -49,7 +49,11 @@
                 <h3>あらすじ</h3>
                 <p class="center" style="line-height: 2em">{!! nl2br($play[$ps]['schema:abstract'][0]) ?: "<span style='color:gray;'>N/A</span>" !!}</p>
             @endif
-            <h3>公演概要</h3>
+            <div style="display: flex; justify-content: flex-start;">
+                <h3 style="margin: 0 10px 0 0">公演概要</h3>
+                <a href="https://twitter.com/share?ref_src=twsrc%5Etfw" class="twitter-share-button" data-lang="ja" data-show-count="false">Tweet</a>
+                <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+            </div>
```

Comment:
h3を入れ子の中に入れてしまうと文書構造がおかしくなってしまうため、あまりよろしくありません。

h3をflexにし、その上でツイートボタンを入れるとHTMLとしてもキレイになりますし、h3の中にaを入れることも、iframeを入れることもコンテンツカテゴリーを逸脱しません。

ついでに、align-items: flex-end; を指定すると、下揃えになって若干良い気がします。

```suggestion
            <h3 style="display: flex; justify-content: flex-start; align-items: flex-end">
                <span style="margin-right: 10px">公演概要</span>
                <a href="https://twitter.com/share?ref_src=twsrc%5Etfw" class="twitter-share-button" data-lang="ja" data-show-count="false">Tweet</a>
                <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
            </h3>
```